jenkins-integration/beaker/install/shared/stop_sut_background_scripts.rb

```diff
@@ -0,0 +1,12 @@
+require 'json'
+
+step "Stop background scripts on SUT" do
+  pids = JSON.parse(File.read("bg_pids.json"))
+
+  Beaker::Log.notify("Stopping #{pids.count} background scripts")
+  Beaker::Log.notify(JSON.pretty_generate(pids))
+  pids.each do |k, v|
+    Beaker::Log.notify("Stopping script '#{k}' with pid '#{v}'")
+    on(master, "kill #{v}")
```

Comment:
Not necessarily opposed to this, but do you have a concrete example in mind of a case where it would be problematic to move on without that?  Also did you test that line that you pasted above?  Or just guessing?